Documentation/platforms/risc-v/litex/boards/arty_a7/index.rst

```diff
@@ -58,8 +58,8 @@ You will need to download the tool-chain for this board:
 
    $ curl https://static.dev.sifive.com/dev-tools/riscv64-unknown-elf-gcc-8.3.0-2019.08.0-x86_64-linux-ubuntu14.tar.gz
 
-Flashing
-========
+vexriscv Flashing
+=================
```

Comment:
Flashing, Configurations, etc are standard in the Documentation/, I suggest keeping this way, since Flashing is alread supposed to be for vexrisc softcore

Reply:
How should we specify the different cores in the documentation?  Should there be an overall section per core?  And then Flashing, Configuration, etc for each core would be under that?

(As an FPGA, what core you run is a bit orthogonal to the board obviously!)

Reply:
I have pushed an updated commit to this PR that introduce a new section level - it looks reasonable enough and keeps the original section names.  Thanks!